server/main.lua

```diff
@@ -39,12 +39,35 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleowner = GetVehicleOwner(plate)
+		local bolo, title, boloid = GetBoloStatus(plate, title, boloid)
+		local warrant, owner, incidentid = GetWarrantStatus(plate, owner, incidentid)
+		local driverslicense = PlayerData.metadata['licences'].driver

```

Comment:
nit camelCase all of these variables: vehicleOwner, boloId, incidentId, driversLicense

Reply:
Amended code in new commit d13ea57